### PR TITLE
Caret selection

### PR DESCRIPTION
**Pull request by greg-kargin**

### CODE REVIEW

File: src/cljc/slurper/text.cljc [hunk 1]
```diff
@@ -110,21 +110,22 @@
           (when (some? i)
```
Comment (greg-kargin) [inline src/cljc/slurper/text.cljc:110]: Not sure why this changes are in PR,  they were in previously
https://github.com/JetBrains/andel/commit/257d13f5821528daf35cbf5f98278537b16f6d91

Comment (zajac) [reply src/cljc/slurper/text.cljc:110]: these are my changes. you got it from merge, not sure why do we see it here.
